src/lava/magma/compiler/compiler.py

```diff
@@ -496,6 +554,26 @@ def _get_channel_type(src: ty.Type[AbstractProcessModel],
                 f"'({src.__name__}, {dst.__name__})' yet."
             )
 
+    @staticmethod
+    def _get_port_dtype(port: AbstractPort,
+                        proc_model: ty.Type[AbstractProcessModel]) -> type:
+        """Returns the type of a port, as specified in the corresponding
```

Comment:
A bit imprecise... Returns the d_type of a Process Port, as specified in the corresponding PortImplementation of the ProcessModel implementing the Process.

Reply:
Done